packages/p2p/src/components/my-ads/ad-status.scss

```diff
@@ -1,10 +1,13 @@
+@mixin ad-status-base {
+    border-radius: 1.6rem;
+    padding: 0.2rem 1.6rem;
+    width: 8.8rem;
+}
```

Comment:
is it fine to have the mixins here or should we add this to the mixins.scss?

Reply:
In my opinion, not really since this mixin is only included here for reducing code duplication and is only specific for ad status styling, since both the active and inactive ad status share a lot of the same css properties. If maybe however in the future we have mixins that is reusable like making calculations etc, we could include it in our own `mixins.scss` file for P2P